shuup/tasks/templates/shuup/admin/tasks/task_edit.jinja

```diff
@@ -44,9 +44,10 @@
                                     </thead>
                                     <tbody>
                                         {% for comment in comments %}
+                                        {% set url = shuup_admin.model_url(comment.author) %}
                                         <tr>
                                             <td class="col-xs-2">{{ comment.created_on|datetime(format="short") }}</td>
-                                            <td class="col-xs-2">{{ comment.author }}</td>
+                                            <td class="col-xs-2"><a href="{{ url }}">{{ comment.author.id }}</a></td>
```

Comment:
is there any purpose to show the customer id instead of the name?

Reply:
yes, register from front -> go to dashboard -> request data anonymization. 

TL;DR; not all contacts have name :)

I can change it so that if name exists, we show that and if not, we show id (like on the Picotable this PR)